ui/src/project.vue

```diff
@@ -834,4 +852,10 @@ padding: 5px 10px;
         margin-right: 20px;
     }
 }
+OL LI::marker {
+    font-size: 2em;
+}
+.capitalize {
```

Comment:
Do you really need to capitalize? Also, I don't like using css class that's named after the type of style applied. You should just use inline css for that, or make the class name to be something like "paper-title".